Shake/Main.lean

```diff
@@ -457,15 +457,19 @@ def main (args : List String) : IO UInt32 := do
     pure (some (path.parent.get! / "scripts" / "noshake.json"))
   else pure none
 
-  -- Read the config file
-  let cfg ← if let some file := cfgFile then
+  -- Read the config file, `validCfgFile? = false` only of the config file is present and invalid
```

Comment:
```suggestion
  -- Read the config file, `validCfgFile? = false` only if the config file is present and valid
```

Reply:
Oh wait, I completely misunderstood what the comment was trying to say.

Reply:
Thanks!  I am glad someone is paying attention to the comments!  :smile:

Reply:
I did go over this after pushing your suggestion: I think that it became correct the second time, but I have nevertheless split the sentence in two.